README.md

```diff
@@ -19,8 +19,8 @@ Icon font with helper classes, and resolution-independent SVG icons to load via
 There are multiple ways to use the icons:
 
 1. Using the CSS helper classes
-2. Extending the predefined Sass placeholders into your own CSS classes
-3. Resolution independent SVGs, using the [responsive image service](http://image.webservices.ft.com/)
+1. Extending the predefined Sass placeholders into your own CSS classes
+1. Resolution independent SVGs, using the [responsive image service](http://image.webservices.ft.com/)
```

Comment:
This is a markdown-valid way of making numbered lists that means you don't need to re-number everything if you remove an item from the list. p cool huh?